src/warc2zim/converter.py

```diff
@@ -388,15 +407,46 @@ def run(self):
 
         self.creator.finish()
 
+    def extract_warcinfo(self, record):
+        """Extract the software value from a warcinfo record"""
+        if self.warc_software:
+            logger.debug("warc_software already set, ignoring warcinfo record")
+            return
+        if get_record_mime_type(record) != "application/warc-fields":
+            logger.warning(
+                f"Unsupported warcinfo record found: {get_record_mime_type(record)}"
+            )
+            return
+        for warcfield in get_record_content(record).decode("UTF-8").splitlines():
+            name_and_value = warcfield.split(":")
```

Comment:
I believe the following is as good (worst case is value contains additional stuff) yet clearer
```suggestion
            name, value = warcfield.split(":", 1)
            if name.strip().lower() != "software":
                continue
            self.warc_software = value.strip()
```